pkg/cluster/kubeclient/status.go

```diff
@@ -0,0 +1,22 @@
+package kubeclient
+
+import (
+	"context"
+	"encoding/json"
+
+	"k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func (u *kubeclient) GetPodsStatus(ctx context.Context, namespaces []string) ([]byte, error) {
+	var pods []*v1.PodList
```

Comment:
var pods []v1.Pod

Reply:
I used `v1.PodList` because that is what MSFT is using/doing. I would also prefer to gather a list of `Pod`s and not a list of `PodList`s